### PR TITLE
Improve webgl-draw-buffers.html

### PR DESCRIPTION
**Pull request by zhenyao**

### CODE REVIEW

**Comment by zhenyao** [comment]
@kenrussell Please take a look.

File: sdk/tests/conformance/extensions/webgl-draw-buffers.html [hunk 1]
```diff
@@ -553,17 +560,30 @@
   if (maxUsable > 1) {
     var bufs1 = makeColorAttachmentArray(maxUsable);
     var bufs2 = makeColorAttachmentArray(maxUsable);
+    var bufs3 = makeColorAttachmentArray(maxUsable);
+    var bufs4 = makeColorAttachmentArray(maxUsable);
```
Comment (kenrussell) [inline sdk/tests/conformance/extensions/webgl-draw-buffers.html]: How about a one-line comment before each "bufs" variable saying what each intends to test? (For example, "First half of color buffers disabled", "Second half", etc.)

**Comment by kenrussell** [comment]
LGTM. One small comment. Please merge yourself. Thanks.

File: sdk/tests/conformance/extensions/webgl-draw-buffers.html (hunk 1 above)
Comment (zhenyao) [inline sdk/tests/conformance/extensions/webgl-draw-buffers.html]: Done.

**Pull request merged**